src/cmds/orgs:create.ts

```diff
@@ -46,9 +47,9 @@ export async function handler(argv: {
 
     console.log(orgsMessage);
   } catch (e) {
+    const errorMessage = `ERROR! Failed to create organizations.\nTry running with \`DEBUG=snyk* <command> for more info\`.\nERROR: ${e.message}`;
     debug('Failed to create organizations.\n' + e);
-    console.error(
-      `${e}.\nTry running with \`DEBUG=snyk* <command> for more info\`.\nERROR: ${e}`,
-    );
+    console.error(errorMessage);
+    setTimeout(() => yargs.exit(1, new Error(errorMessage)), 3000);
```

Comment:
needed to le bunyan finish writing logs, there is no way to await completion and exit code 1 kills the process which results in unfinished logs.